tests/test_imports.py

```diff
@@ -6,3 +6,4 @@ def test_basic(self):
         import bq_helper
         import cleverhans
         import tensorflow_gcs_config
+        import tensorflow_datasets
```

Comment:
This package downloads and formats curated datasets and a nice format and have a few options for shuffling the data and so on.

We can't smoke test the `.load(...)` method since it is making a network call to GCS. Testing that import do not throw an error is the best we can do.